lib/MutationPoint.cpp

```diff
@@ -127,33 +137,47 @@ std::string MutationPoint::getMutatorIdentifier() const {
   return mutator->getUniqueIdentifier();
 }
 
-const std::string &MutationPoint::getDiagnostics() { return diagnostics; }
+const std::string &MutationPoint::getDiagnostics() {
+  return diagnostics;
+}
 
-const std::string &MutationPoint::getDiagnostics() const { return diagnostics; }
+const std::string &MutationPoint::getDiagnostics() const {
+  return diagnostics;
+}
 
-const std::string &MutationPoint::getReplacement() { return replacement; }
+const std::string &MutationPoint::getReplacement() {
+  return replacement;
+}
 
 const SourceLocation &MutationPoint::getSourceLocation() const {
   return sourceLocation;
 }
 
-Function *MutationPoint::getOriginalFunction() { return originalFunction; }
+Function *MutationPoint::getOriginalFunction() {
+  return originalFunction;
+}
 
 void MutationPoint::setMutatedFunction(llvm::Function *function) {
   function->setName(getMutatedFunctionName());
   this->mutatedFunction = function;
 }
 
 std::string MutationPoint::getTrampolineName() {
-  return originalFunction->getName().str() + "_" +
-         bitcode->getUniqueIdentifier() + "_trampoline";
+  return originalFunction->getName().str() + "_" + bitcode->getUniqueIdentifier() + "_trampoline";
 }
 
 std::string MutationPoint::getMutatedFunctionName() {
   return getUniqueIdentifier();
 }
 
 std::string MutationPoint::getOriginalFunctionName() const {
-  return originalFunction->getName().str() + "_" +
-         bitcode->getUniqueIdentifier() + "_original";
+  return originalFunction->getName().str() + "_" + bitcode->getUniqueIdentifier() + "_original";
+}
+
+std::string MutationPoint::dump() const {
+  std::stringstream ss;
+  ss << "Mutation Point: " << getMutator()->getUniqueIdentifier() << " "
+     << getSourceLocation().filePath << ":" << getSourceLocation().line << ":"
+     << getSourceLocation().column;
```

Comment:
Could've been `getSourceLocation().dump()`, right? ;)

Reply:
Good idea. I am all in for having focused `.dump()` methods.